### ISSUE TITLE
Directory layout thought

### ISSUE THREAD

**Issue by JohnHadish** [SystemsGenetics/GEMmaker#135]
Just an idea on GEMmaker's directory layout. I was thinking we should consider numbering the important directories to make it clearer for users what needs to be done. It would not be all of them, but something like this:

```
00_examples
01_input
02_output
bin
docker
docs
files
images
LICENSE.txt
main-e2e.nf
main.nf
multiqc_config.yaml
nextflow.config
README.md
requirements.txt
templates
tests
```
This is related to issue https://github.com/SystemsGenetics/GEMmaker/issues/115

**Comment by spficklin** [comment]
Lowest priority for a v1.1 release.

**Comment by spficklin** [comment]
How about we push this to v1.2 to give us time to think more on it?

**Comment by JohnHadish** [comment]
Additional thought:
People are also getting confused by why the `reference` is in the `input` direcctory. I think we should make it it's own directory and rename input. Could look like this:
```
00_examples
01_reference
02_input_data
03_output
bin
docker
docs
files
images
LICENSE.txt
main-e2e.nf
main.nf
multiqc_config.yaml
nextflow.config
README.md
requirements.txt
templates
tests
```

**Comment by spficklin** [comment]
Agreed.

**Comment by spficklin** [comment]
I'm changing this to a v1.1 milestone because people are getting confused by the directory structure by forgetting to remove sample files or not fully understanding what the paths should be because multiple variables are needed to build a full path and that's just not very obvious.

**Comment by JohnHadish** [comment]
> Additional thought:
> People are also getting confused by why the `reference` is in the `input` direcctory. I think we should make it it's own directory and rename input. Could look like this:
> 
> ```
> 00_examples
> 01_reference
> 02_input_data
> 03_output
> bin
> docker
> docs
> files
> images
> LICENSE.txt
> main-e2e.nf
> main.nf
> multiqc_config.yaml
> nextflow.config
> README.md
> requirements.txt
> templates
> tests
> ```

Just got tripped up by this again. It is the little aesthetics that make a program useful.

**Comment by spficklin** [comment]
I agree, we need to clean up some things. I think we can do this after the next major release.

**Comment by spficklin** [comment]
Fixed with PR #205.  Users can now keep their fastq files where they want and it easier to indicate where they are in the config file.  There is no longer an `input` folder. All of the demo data has been moved to a `demo` folder.
